docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contribute to Stryker4s
+
+This is the contribution guide for Stryker4s. Great to have you here! Here are a few ways you can help to make this project better.
+
+## Adding a new feature
+
+New features are welcome! Both as ideas or in the form of a pull request.
+
+1. Please [create an issue](https://github.com/stryker-mutator/stryker4s/issues/new) with your idea first or let us know via [Gitter](https://gitter.im/stryker-mutator/stryker4s).
+2. Create a fork on your GitHub account.
+3. When writing your code, please conform the existing coding style. This can be as easy as running `sbt scalafmt` to format your code. Or take a moment to review the [Scala style guide](https://docs.scala-lang.org/style/).
```

Comment:
Or have the latest version of intellij and configure it to use .scalafmt

Reply:
Added. The scalafmt documentation isn't wholly up to date anymore as intellij now has scalafmt support built in and doesn't need the plug-in anymore